app/jobs/participant_outcomes/batch_send_latest_outcomes_job.rb

```diff
@@ -1,25 +1,27 @@
 # frozen_string_literal: true
 
+require Rails.root.join "lib/active_job/queue_adapters/sidekiq_adapter_patch"
+
 module ParticipantOutcomes
   class BatchSendLatestOutcomesJob < ApplicationJob
     queue_as :participant_outcomes
     sidekiq_options retry: false
 
     DEFAULT_BATCH_SIZE = 200
-    REQUEUE_DELAY = 2.minutes
+    DEFAULT_REQUEUE_DELAY = 2.minutes
 
-    def perform(batch_size = DEFAULT_BATCH_SIZE)
+    def perform(batch_size = DEFAULT_BATCH_SIZE, delay = DEFAULT_REQUEUE_DELAY)
       unless can_enqueue_jobs?
         Rails.logger.info "BatchSendLatestOutcomesJob: Unable to proceed due to pending SendToQualifiedTeachersApiJob jobs; re-queueing to try again."
-        self.class.set(wait: REQUEUE_DELAY).perform_later
+        self.class.set(wait: delay).perform_later
         return
       end
 
       @batch_size = batch_size
 
-      outcomes.first(@batch_size).each { |outcome| SendToQualifiedTeachersApiJob.perform_later(outcome.id) }
+      outcomes.first(@batch_size).each { |outcome| SendToQualifiedTeachersApiJob.perform_later(participant_outcome_id: outcome.id) }
 
-      self.class.set(wait: REQUEUE_DELAY).perform_later if should_enqueue_again?
+      self.class.set(wait: delay).perform_later if should_enqueue_again?
```

Comment:
could we pass in the delay/batch size to when we're rescheduling this job, as otherwise it will fallback to the old settings?